modules/spring-config/src/main/java/org/mule/runtime/config/internal/SpringXmlConfigurationBuilder.java

```diff
@@ -58,8 +58,8 @@
 
   private ApplicationContext parentContext;
   private MuleArtifactContext muleArtifactContext;
-  private ArtifactType artifactType;
-  private ComponentBuildingDefinitionProvider runtimeComponentBuildingDefinitionsProvider;
+  private final ArtifactType artifactType;
+  private final ComponentBuildingDefinitionProvider runtimeComponentBuildingDefinitionsProvider;
```

Comment:
For this two fields that are changed to a **final** modifier, isn't ir required to set them in the static constructor?

Reply:
it is being set in the constructor. The `final` is added automatically by the IDE when it detects that  it is not being written anywhere else.